cli/common.go

```diff
@@ -18,3 +18,53 @@ type CommonFlags struct {
 	TLSOptions *tlsconfig.Options
 	TrustKey   string
 }
+
+// Command is the struct contains command name and description
+type Command struct {
+	Name        string
+	Description string
+}
+
+// DockerCommands stores all the docker command
+var DockerCommands = map[string]Command{
+	"attach":  {"attach", "Attach to a running container"},
```

Comment:
I kinda agree with @duglin, would be really cool if we didn't have this duplication :sweat_smile: